app/controllers/admin/conferences_controller.rb

```diff
@@ -82,7 +82,7 @@ def update
       short_title = @conference.short_title
       @conference.assign_attributes(conference_params)
       send_mail_on_conf_update = @conference.notify_on_dates_changed?
-      delete_event_schedules if @conference.start_hour_changed? || @conference.end_hour_changed?
+      delete_event_schedules if @conference.valid? && (@conference.start_hour_changed? || @conference.end_hour_changed?)
```

Comment:
you could move this inside the

`if @conference.update_attributes(conference_params)`

and then you now the conference is valid and you can just write:

`delete_event_schedules if @conference.start_hour_changed? || @conference.end_hour_changed?`

I like it more as you are sure that the conference start/end time was changed.